usecases/config/config_handler.go

```diff
@@ -116,6 +116,7 @@ type Config struct {
 	ForceFullReplicasSearch             bool                     `json:"force_full_replicas_search" yaml:"force_full_replicas_search"`
 	RecountPropertiesAtStartup          bool                     `json:"recount_properties_at_startup" yaml:"recount_properties_at_startup"`
 	ReindexSetToRoaringsetAtStartup     bool                     `json:"reindex_set_to_roaringset_at_startup" yaml:"reindex_set_to_roaringset_at_startup"`
+	ReindexCorruptedIndexesAtStartup    map[string][]string      `json:"reindex_corrupted_indexes_at_startup" yaml:"reindex_corrupted_indexes_at_startup"`
```

Comment:
nit: may be we should call it just `ReindexIndexesAtStartup` (without `corrupt` everywhere).

Reply:
nit2: Also would be nice to have some comment explaning what this `map[string][]string` means? From the changes it looks like `map[class] -> []properties`?. Does the `class` name should be GQL form?

Reply:
renamed, added comment to the field